src/Entity/ServiceProvider.php

```diff
@@ -106,6 +106,20 @@ class ServiceProvider extends AbstractRole
     public $requestedAttributes;
 
     /**
+     * @var null|string
+     *
+     * @ORM\Column(name="url_en", type="string")
+     */
+    public $urlEn;
```

Comment:
I'm not sure. I like descriptive and accurate names, but the other properties also have names that map to Janus' way of naming things. 

This creates a certain expectation I would like to fulfil, but I would use a more meaningful name in the application that uses the data (not sure if this would be `EngineBlock` or, in my case, `Profile`).